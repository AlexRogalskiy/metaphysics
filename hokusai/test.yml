---
version: '2'
services:
  metaphysics:
    command: yarn test
    extends:
      file: common.yml
      service: metaphysics
    environment:
    - NODE_ENV=test
    env_file: ../.env.test
