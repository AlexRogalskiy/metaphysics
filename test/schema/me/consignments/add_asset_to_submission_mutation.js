import { runAuthenticatedQuery } from "test/utils"

describe("addAssetToConsignmentSubmission", () => {
  it("creates a submission and returns its new data payload", () => {
    const mutation = `
      mutation {
        addAssetToConsignmentSubmission(input:
          { asset_type: "image", gemini_token: "12345", submission_id: "123", clientMutationId: "123" }
        ){
          asset {
            submission_id
            gemini_token
          }
        }
      }
    `

    const rootValue = {
      assetCreateLoader: () =>
        Promise.resolve({
          id: "106",
          gemini_token: "12345",
          submission_id: "123",
        }),
    }

    return runAuthenticatedQuery(mutation, rootValue).then(({ assetCreateLoader }) => {
      expect(assetCreateLoader).toMatchSnapshot()
    })
  })
})
