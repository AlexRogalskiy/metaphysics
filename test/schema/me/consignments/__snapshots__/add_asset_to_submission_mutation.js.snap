// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`addAssetToConsignmentSubmission creates a submission and returns its new data payload 1`] = `undefined`;
